src/css/app.scss

```
@@ -8,3 +8,23 @@ code {
   margin-right: 3px;
   border-radius: 3px;
 }
+
+.file-status-staged {
+  color: #21BA45;
+}
+
+.file-status-unstaged {
+  color: #C10015;
+}
+
+.file-status-untracked {
+  color: #C10015;
+}
```

If it's the same color :
```suggestion
.file-status-unstaged, .file-status-untracked {
  color: #C10015;
}
```

---

Yes but in case of we want to set different color I prefere to separate the definition